src/bin/Makefile

```diff
@@ -32,6 +32,13 @@ ADDLIBS = ../decoder/kaldi-decoder.a ../lat/kaldi-lat.a ../lm/kaldi-lm.a \
           ../tree/kaldi-tree.a ../util/kaldi-util.a \
           ../matrix/kaldi-matrix.a ../base/kaldi-base.a
 
+ifeq ($(CUDA), true)
+LDFLAGS += $(CUDA_LDFLAGS)
+LDLIBS += $(CUDA_LDLIBS)
+ADDLIBS += ../cudamatrix/kaldi-cudamatrix.a
+BINFILES += latgen-faster-mapped-cuda decode-faster-mapped-cuda
```

Comment:
Any plans to make a decoder that takes an nnet3 model directly and produces posteriors from featurized inputs?

Reply:
Yes. Hang is helping me to do this recently.